client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -17,95 +17,94 @@ export const MAX_POSTS_TO_SHOW = 6;
-						<Path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-4.86 8.86l-3 3.87L9 13.14 6 17h12l-3.86-5.14z" />
-					</SVG>
-				</figure>
+						<SVG
+							className="jp-related-posts-i2__post-image-placeholder-figure-square"
+							xmlns="http://www.w3.org/2000/svg"
+							width="100%"
+							height="100%"
+							viewBox="0 0 350 200"
+						>
+							<title>{ __( 'Grey square' ) }</title>
+							<Path d="M0 0h350v200H0z" fill="#8B8B96" fill-opacity=".1" />
+						</SVG>
+						<SVG
+							className="jp-related-posts-i2__post-image-placeholder-figure-icon"
+							xmlns="http://www.w3.org/2000/svg"
+							width="24"
+							height="24"
+							viewBox="0 0 24 24"
+						>
+							<title>{ __( 'Icon for image' ) }</title>
+							<Path fill="none" d="M0 0h24v24H0V0z" />
+							<Path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-4.86 8.86l-3 3.87L9 13.14 6 17h12l-3.86-5.14z" />
+						</SVG>
+					</figure>
+				</li>
 			) }
 
 			{ props.displayDate && (
-				<div className="jp-related-posts-i2__post-date has-small-font-size">
-					{ __( 'August 3, 2018' ) }
-				</div>
+				<div className="jp-related-posts-i2__post-date">{ __( 'August 3, 2018' ) }</div>
 			) }
 			{ props.displayContext && (
-				<div className="jp-related-posts-i2__post-context has-small-font-size">
-					{ __( 'In “Uncategorized”' ) }
-				</div>
+				<div className="jp-related-posts-i2__post-context">{ __( 'In “Uncategorized”' ) }</div>
 			) }
-		</div>
+		</ul>
 	);
 }
 
 function RelatedPostsEditItem( props ) {
 	return (
-		<div
+		<ul
 			className="jp-related-posts-i2__post"
 			id={ props.id }
 			aria-labelledby={ props.id + '-heading' }
 		>
-			<a
-				className="jp-related-posts-i2__post-link"
-				id={ props.id + '-heading' }
-				href={ props.post.url }
-				rel="nofollow noopener noreferrer"
-				target="_blank"
-			>
-				{ props.post.title }
-			</a>
-			{ props.displayThumbnails && props.post.img && props.post.img.src && (
-				<a className="jp-related-posts-i2__post-img-link" href={ props.post.url }>
-					<img
-						className="jp-related-posts-i2__post-img"
-						src={ props.post.img.src }
-						alt={ props.post.title }
-						rel="nofollow noopener noreferrer"
-						target="_blank"
-					/>
+			<li className="jp-related-posts-i2__post-link">
+				<a
+					id={ props.id + '-heading' }
+					href={ props.post.url }
+					rel="nofollow noopener noreferrer"
+					target="_blank"
+				>
+					{ props.post.title }
 				</a>
+			</li>
+			{ props.displayThumbnails && props.post.img && props.post.img.src && (
+				<li className="jp-related-posts-i2__post-img-link">
+					<a href={ props.post.url } rel="nofollow">
+						<img
+							className="jp-related-posts-i2__post-img"
+							src={ props.post.img.src }
+							alt={ props.post.img.alt_text }
+							rel="nofollow noopener noreferrer"
+							target="_blank"
```

Comment:
These `rel` and `target` were prolly meant to be in `a` tag, not in `img`?

Reply:
Yes! Thank you!